### PR TITLE
Sandbox Sql fixture

### PR DESCRIPTION
**Pull request by gaboraranyossy-da**
- introduces a new SandboxSQL fixture to be used with ledger api integration tests
- also fixes a few issues came out on running integration tests with the SandboxSQL fixture.

Note that not all integration tests would pass with this new fixture, so right now it's disabled. The goal is to fix all issues in follow-up PRs and enable the fixture and general SQL support as a final step.

### CODE REVIEW

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/LedgerDao.scala [hunk 1]
```diff
@@ -59,9 +65,11 @@ trait LedgerDao {
     * @param offset the offset to look at
     * @return the LedgerEntry found, or throws an exception
     */
-  def lookupLedgerEntryAssert(offset: Long): Future[LedgerEntry] =
+  def lookupLedgerEntryAssert(offset: Long): Future[LedgerEntry] = {
+    println(s"reading record with offset: $offset")
```
Comment (rautenrieth-da) [inline ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/LedgerDao.scala]: comment out this line?

**Comment by rautenrieth-da** [comment]
A couple of questions:
* Description says _"introduces a new SandboxSQL fixture"_. There is already a `SandboxFixtureSQL.scala` on master, which I assume should be deleted in favor of the `MultiLedgerFixture` updated in this PR?
* This PR adds a `CommandTransactionSQLIT.scala` test. This is only temporary, right? As far as I understood, all ITs should be updated to use the `MultiLedgerFixture` and automatically run on both the in-memory and the SQL ledger.

**Comment by gaboraranyossy-da** [comment]
Thanks for your comments @rautenrieth-da. You are absolutely right. They got on this PR as I was using another PR as base from which these classes got removed eventually. I remove them.

File: ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/LedgerDao.scala (hunk 1 above)
Comment (gaboraranyossy-da) [reply ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/LedgerDao.scala]: yes, debug residue

**Comment by gaboraranyossy-da** [comment]
retest this please

**Pull request merged**